qa/os/src/test/java/org/elasticsearch/packaging/test/PackageTests.java

```diff
@@ -374,4 +340,58 @@ public void test83serviceFileSetsLimits() throws Exception {
 
         stopElasticsearch(sh);
     }
+
+    public void test90DoNotCloseStderrWhenQuiet() throws Exception {
```

Comment:
I don't think this tests what we want to test since the JVM never starts here (i.e., Elasticsearch code never runs). Instead, you could create an invalid `elasticsearch.yml`.

Reply:
I know how this happened 😅 I was trying to trigger the bootstrap check about using the serial collector, and got in a muddle. I'll rework it.

Reply:
Take 2 ready for a look @jasontedor.